django/urls/resolvers.py

```diff
@@ -273,9 +273,9 @@ def check(self):
         route = self._route
         if '(?P<' in route or route.startswith('^') or route.endswith('$'):
             warnings.append(Warning(
-                "Your URL pattern {} has a route that contains '(?P<', begins "
-                "with a '^', or ends with a '$'. This was likely an oversight "
-                "when migrating to django.urls.path().".format(self.describe()),
+                "Your URL pattern {} has a route that contains begins with a "
+                "'^' or ends with a '$'. This was likely an oversight when "
+                "migrating to django.urls.path().".format(self.describe()),
```

Comment:
I will revert this changes. Test failure is related with invalid identifier, if we changed `named-group` to the `named_group` all works as expected and the current version of this warning is still valid.

Reply:
You're right!